spec/api-app-spec.js

```diff
@@ -302,31 +303,31 @@ describe('app module', () => {
 
     afterEach(() => closeWindow(w).then(() => { w = null }))
 
-    it('can import certificate into platform cert store', (done) => {
-      let options = {
+    it('can import certificate into platform cert store', done => {
+      const options = {
         certificate: path.join(certPath, 'client.p12'),
         password: 'electron'
       }
 
       w = new BrowserWindow({ show: false })
 
       w.webContents.on('did-finish-load', () => {
-        assert.equal(w.webContents.getTitle(), 'authorized')
+        expect(w.webContents.getTitle()).to.equal('authorized')
         done()
       })
 
       ipcRenderer.once('select-client-certificate', (event, webContentsId, list) => {
-        assert.equal(webContentsId, w.webContents.id)
-        assert.equal(list.length, 1)
-        assert.equal(list[0].issuerName, 'Intermediate CA')
-        assert.equal(list[0].subjectName, 'Client Cert')
-        assert.equal(list[0].issuer.commonName, 'Intermediate CA')
-        assert.equal(list[0].subject.commonName, 'Client Cert')
+        expect(webContentsId).to.equal(w.webContents.id)
+        expect(list.length).to.equal(1)
+        expect(list[0].issuerName).to.equal('Intermediate CA')
+        expect(list[0].subjectName).to.equal('Client Cert')
+        expect(list[0].issuer.commonName).to.equal('Intermediate CA')
+        expect(list[0].subject.commonName).to.equal('Client Cert')
```

Comment:
```js
expect(list[0]).to.deep.equal({
 issuerName: 'Intermediate CA',
 <...>
})
```

We can use `include` instead of `equal` here if there're some other fields we don't test now.